### PR TITLE
[luci/import] Revise importVerifyModule

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise importVerifyModule to verify only when file size is less than 2G.

### CODE REVIEW

**Comment by seanshpark** [comment]
- for https://github.com/Samsung/ONE/issues/13674#issuecomment-2370665768

**Review by jinevening — APPROVED** [review]
LGTM

File: compiler/luci/import/src/ImporterEx.cpp [hunk 1]
```diff
@@ -43,11 +51,14 @@ std::unique_ptr<Module> ImporterEx::importVerifyModule(const std::string &input_
...
   {
-    std::cerr << "ERROR: Invalid input file '" << input_path << "'" << std::endl;
-    return nullptr;
+    flatbuffers::Verifier verifier{data_data, data_size};
+    if (!circle::VerifyModelBuffer(verifier))
+    {
+      std::cerr << "ERROR: Invalid input file '" << input_path << "'" << std::endl;
+      return nullptr;
+    }
   }
```
Comment (shs-park) [inline compiler/luci/import/src/ImporterEx.cpp:62]: If the size exceeds 2GB, VerifyModelBuffer will probably not be executed. Is that okay?

**Review by shs-park — APPROVED** [review]
I left a question,
PTAL
=)

File: compiler/luci/import/src/ImporterEx.cpp (hunk 1 above)
Comment (seanshpark) [reply compiler/luci/import/src/ImporterEx.cpp:62]: Yes, not executed.
I can't say it's OK but Tensorflow is doing like this.

**Pull request merged**